hi-ml-histopathology/testhisto/testhisto/mocks/tiles_generator.py

```diff
@@ -94,17 +97,27 @@ def create_mock_metadata_dataframe(self) -> pd.DataFrame:
 
         df = pd.DataFrame(data=mock_metadata)
         df.to_csv(os.path.join(self.tmp_path, PandaTilesDataset.DEFAULT_CSV_FILENAME), index=False)
+        self.n_tiles: int = tiles_count
```

Comment:
Isn't `self.n_tiles` already defined? Would it make sense to distinguish tiles per slide from total tiles?

Reply:
This is a bit subtle, cause self.n_tiles is used a bit differently depending on the type of tiles positioning... I should definitely use different parameters for clarity